Instructions for OC Web example in Kubernetes

draffensperger opened the pull request:
This adds instructions for how to run the OpenCensus web initial load example on GKE (Kubernetes on Google Cloud Platform) using Stackdriver trace.

mayurkale22 commented on examples/initial_load/Dockerfile:
```diff
@@ -0,0 +1,31 @@
+COPY ./static/*.js /static/
+COPY --from=build ./root/example_server /
+ENTRYPOINT ["/example_server"]
+EXPOSE 8000 8000
```
Duplicate?

draffensperger replied:
Yes, thanks, removed the extra ` 8000`.

mayurkale22 commented on examples/initial_load/README.md:
```diff
@@ -63,6 +64,84 @@ the trace viewer for the exporter you set up and view the trace of your initial
+`EXTERNAL-IP` column has been filled in. If it is still pending, then wait a bit
+and run it again until it's available.
+
+Once you the agent has an external IP, you can deploy the example service that
```
`s/Once you the agent/Once the agent` ?

draffensperger replied:
Thanks, done.

mayurkale22 commented:
What is the role of `examples/initial_load/static/.gitkeep` file?

draffensperger commented:
The role of `examples/initial_load/static/.gitkeep` was to make it so that the `static` folder would exist in git to be ready to have the built JS files copied into it. But that isn't strictly needed and I was able to remove it and instead add a `mkdir -p static`. I also added `static/` to `.gitignore` to prevent the added files from being included (I had that change in an earlier version of this but forgot to get it added when I changed branches)

mayurkale22 commented:
@songy23 for GO related reviews.

songy23 commented on examples/initial_load/Dockerfile:
```diff
@@ -0,0 +1,31 @@
+ADD go.mod .
+ADD go.sum .
+ADD server.go .
+RUN go get -d -v
```
`GO111MODULE=on go get -d -v`?

draffensperger replied:
Done, thanks

songy23 commented on examples/initial_load/Dockerfile:
```diff
@@ -0,0 +1,31 @@
+ADD go.sum .
+ADD server.go .
+RUN go get -d -v
+RUN CGO_ENABLED=0 go build -o example_server
```
Same here.

draffensperger replied:
Done
